docs/notebook-demos/ingesting-an-image.ipynb

```diff
@@ -0,0 +1,198 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Ingesting an Image\n",
+    "\n",
+    "This example shows how to ingest a grayscale image and save the results locally. It is assumed that you have already read through the guides before beginning this tutorial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting the Data\n",
+    "\n",
+    "Before we can begin with the ingest, we must first download the data from S3. This curl command will download a file from S3 and save it to your `/tmp` direcotry.\n",
```

Comment:
would it be beneficial to link to an image that is on a non-azavea public domain?

like maybe just something on earth on AWS?

either way - an explanation of what this dataset they are about to download is would be helpful